vision/product_search/purge_products_in_product_set.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package productsearch contains samples for Google Cloud Vision API Product Search.
+package productsearch
+
+// [START vision_product_search_purge_products_in_product_set]
+
```

Comment:
Delete newline between, otherwise this shows up when embedded.

Reply:
Sure, done